enmasse/ansible/playbooks/openshift/workshop.retry

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
@ppatierno This file should not be committed.

Reply:
@lulf sorry, I have just deleted it and excluded using the .gitignore for avoiding a future mistake.